### DISCUSSION TITLE
CSR request not passing through the Distinguished Name properties

### DISCUSSION THREAD

**Discussion by b-rad086** [openssl/openssl#25667] [Q&A] [answered]
Hi,
I have been trying to generate a CSR using the below command but it's not passing through the DN information from the CNF file
"openssl req -new -subj "/" -config request.cnf -keyout request.key -out request.csr"

Any ideas on whats missing to get the DN details to passthrough?

OpenSSL version - 3.3.2.0

[request.txt](https://github.com/user-attachments/files/17336566/request.txt)
[request_cnf.txt](https://github.com/user-attachments/files/17336603/request_cnf.txt)

**Comment by gingerbeard-man** [comment, marked as answer]
Simply drop the `-subj "/"` which is overriding the values from the config file with an empty DN.

`openssl req -new -config request.cnf -keyout request.key -out request.csr`

I used to have the opposite problem with OpenSSL 1.0.2 - it would ignore the -subj parameter and always use the distinguished_name from the config file.  Nice to see this now works with version 3.x.

**Reply by b-rad086** [reply]
This looks like it has worked, thankyou ;-)
